src/uk/gov/defra/ffc/Build.groovy

```diff
@@ -108,4 +109,19 @@ class Build implements Serializable {
         }
       }
   }
+
+  static void triggerMultiBranchBuilds(def ctx, String defaultBranch) {
+    String jobPath = ctx.JOB_NAME
+    String multiBranchJob = jobPath.substring(0, jobPath.lastIndexOf('/'))
+    def item = Jenkins.get().getItemByFullName(multiBranchJob)
+    def jobs = item.allJobs.collect { it }
+    item = null // CPS -- remove reference to non-serializable object
+    for (job in jobs) {
+      String branchName = job.fullName.substring(job.fullName.lastIndexOf('/') + 1)
+      if (branchName != defaultBranch && branchName != ctx.BRANCH_NAME) {
```

Comment:
To avoid endless loop, do not run on default branch or the branch that triggers the trigger.  Should always be default branch but just in case we have a scenario where it's not.